base/combinatorics.jl

```diff
@@ -1,5 +1,12 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
+module Combinatorics
```

Comment:
I've just checked and it doesn't seem to cause more conflicts than it already does, i.e. by exporting function names already exported in `Base`.